javascript/javascript1/week1/index.js

```diff
@@ -23,3 +23,43 @@ if (shouldShowResultInDogYears) {
 else {
     console.log('Your dog will be ' + dogYear + ' human years old in ' + dogYearFuture);
 }
+
+// Housey pricey (A house price estimator)
+
+
+// Peter Hous
+
+const petersWide = 8;
+const petersDeep = 10;
+const petersHigh = 10;
+const peterGardenSizeInM2 = 100,
+
+const volumeInMeters = petersWide * petersDeep * petersHigh;
+const housePrice = volumeInMeters * 2.5 * 1000 + peterGardenSizeInM2 * 300;
+
+const peterHousCost = 2.500.000;
```

Comment:
To represent the number `2,500,000` in JavaScript, you should write it as `2500000` without any dots or commas to separate the thousands. Otherwise, JS will interpret it as `2.5`. Try removing the dots🙂